projects/demo/src/modules/components/line-chart/index.html

```diff
@@ -7,31 +7,29 @@
         <tui-doc-example
             id="line"
             heading="Line"
-            [content]="example1"
-        >
-            <tui-line-chart-example-1 />
-        </tui-doc-example>
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample"
+        />
 
         <tui-doc-example
             id="curve"
             heading="Smooth"
-            [content]="example2"
-        >
-            <tui-line-chart-example-2 />
-        </tui-doc-example>
+            [component]="2 | tuiComponent"
+            [content]="2 | tuiExample"
+        />
 
         <tui-doc-example
             id="dash"
             heading="Dotted"
-            [content]="example3"
-        >
-            <tui-line-chart-example-3 />
-        </tui-doc-example>
+            [component]="3 | tuiComponent"
+            [content]="3 | tuiExample"
+        />
 
         <tui-doc-example
             id="hint"
             heading="Hint"
-            [content]="example4"
+            [component]="4 | tuiComponent"
+            [content]="4 | tuiExample"
         >
             <tui-notification class="tui-space_bottom-6">
```

Comment:
I don't think we need it now that all examples are standalone. We can also do ngFor over the examples

Reply:
fixed